lua/toggleterm/ui.lua

```diff
@@ -183,13 +183,27 @@ local split_commands = {
   },
 }
 
+--- @param size number/function
+local function resolve_size(size)
+  if type(size) == 'number' then
+    return function()
```

Comment:
It seems a bit pointless for this to return a function that will just return a number,
could this be something like
```lua
function resolve_size(size, term)
if type(size) == 'number' then
  return size
end
if term and type(size) == 'function' then
 return size(term)
end
```